backend/data/skills.yaml

```diff
@@ -424,23 +504,27 @@ requirements:
       min: 4
     Logistics Cruisers:
       min: 5
-    Remote Armor Repair Systems:
-      min: 5
     Radar Sensor Compensation:
       elite: 4
     Amarr Cruiser:
       min: 5
     Light Drone Operation:
-      min: 3
+      min: 1
       elite: 5
     Medium Drone Operation:
-      min: 3
+      min: 1
       elite: 5
+    CPU Management:
+      min: 4
+    Power Grid Management:
+      min: 5
     Cybernetics:
       min: 5
+    Long Range Targeting:
```

Comment:
Skill can be removed from ship specific as should be an all logi requirement imo.